decidim-system/app/views/decidim/system/organizations/new.html.erb

```diff
@@ -6,7 +6,9 @@
 
 <%= decidim_form_for(@form) do |f| %>
   <div class="form__wrapper">
-    <%= f.text_field :name, autofocus: true %>
+    <div>
+      <%= f.translated :text_field, :name %>
```

Comment:
I think the logic it's a bit tricky: as a new organization does not have defined any language, I think we should not show the language selector for new Organizations. It should be a `text_field` as it was before, and then in the edit form show the language tabs for the language enabled for this organization. Can you implement that please?

Reply:
Actually here is a small issue. The Form used here is a `RegisterOrganizationForm` which is being extended from `UpdateOrganizationForm`. This `UpdateOrganizationForm` is actually having and using the translated attribute name. In order to implement your suggestion, we should create a new form that would have most of the elements that exist in `UpdateOrganizationForm`, so that we can add the requested behavior.

Reply:
fixed in [0e9e1fb](https://github.com/decidim/decidim/pull/12681/commits/0e9e1fb8baa742cb3b36498ac78c7647e4776311)